assets/webpack.config.js

```diff
@@ -319,7 +320,7 @@ module.exports = (env, options) => ({
       outputFilename: '../licenses.json',
       licenseOverrides: {
         'janus-script@1.9.2': 'MIT',
-        'phoenix_html@3.2.0': 'MIT',
+        'phoenix_html@3.3.1': 'MIT',
```

Comment:
I noticed a warning related to this license version during assets build, this resolves it